src/config.rs

```diff
@@ -224,8 +230,10 @@ pub fn construct_config(pc: ProxyConfig) -> Result<Config, Error> {
         // TODO: full FindRootCAForXDS logic like in Istio
         xds_root_cert: RootCert::File("./var/run/secrets/istio/root-cert.pem".parse().unwrap()),
         ca_address,
+        custom_signer,
         // TODO: full FindRootCAForCA logic like in Istio
         ca_root_cert: RootCert::File("./var/run/secrets/istio/root-cert.pem".parse().unwrap()),
+        k8s_ca_crt: RootCert::File("./var/run/secrets/kubernetes.io/serviceaccount/ca.crt".parse().unwrap()),
```

Comment:
Is there a better place from where this can be picked up? Are we sure that the service account token corresponding to ztunnel will always be mounted here>

Reply:
yeah it's gonna be there whenever ztunnel will be used on K8s